File: 백준/Bronze/2920. 음계/음계.cc
#include<iostream>
#include<string>
#include<sstream>
#include<vector>

using namespace std;

int main(void){
    int num[8];

    for(int i=0; i<8; i++){
        cin>>num[i];
    }

    bool ascen=true;
    bool descen=true;
    for(int i=0;i<8;i++){
        if(num[i]==i+1&&ascen) {
            ascen=true;
             descen = false;
            }
        else if (num[i]==8-i&&descen)   {
            ascen = false;
            descen=true;
        }
        else {
            ascen = false;
            descen = false;
        }
    }
    if(ascen) cout<<"ascending";
    else if(descen) cout<<"descending";
    else cout<<"mixed";
    
    return 0;
}

File: 백준/Bronze/2920. 음계/음계.py
numlist = list(map(int, input().split()))

asc = True
desc = True
for i in range(7):
    if numlist[i] > numlist[i+1]:
        asc = False
    if numlist[i] < numlist[i+1]:
        desc = False
    if asc == False and desc == False:
        break
if asc:
    print("ascending")
elif desc:
    print("descending")
else:
    print("mixed")
